core/logger/logger.go

```diff
@@ -14,6 +14,9 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+// this permission grants read / write accccess to file owners only
+const readWritePerms = os.FileMode(0600)
```

Comment:
This might be a nitpick, but since this value is used in two places, I would probably move it into the `utils` package.

Reply:
Maybe stick to rule of three for now? I’m concerned also util is not a good place to put it.